packages/cosmic-proto/README.md

```diff
@@ -1,9 +1,280 @@
-# Protobuf Stubs for Cosmic Swingset
+# @agoric/cosmic-proto
 
-This package exports JavaScript and TypeScript generated from Cosmic Swingset
-Protobuf IDL so that clients can communicate with the Agoric chain.
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/545047/188804067-28e67e5e-0214-4449-ab04-2e0c564a6885.svg" width="80"><br />
+    Protobuf interfaces for Agoric on Cosmos
+</p>
 
-Package exports are defined in `exports` and also made available in the
-`swingset/` file path for backwards compatibility.
 
-See also [MAINTAINERS.md](MAINTAINERS.md).
+## install
+
+```sh
+npm install @agoric/cosmic-proto
+```
+## Table of contents
+
+- [Usage](#usage)
+    - [RPC Clients](#rpc-clients)
+    - [Composing Messages](#composing-messages)
+        - Cosmos, CosmWasm, and IBC
+            - [CosmWasm](#cosmwasm-messages)
+            - [IBC](#ibc-messages)
+            - [Cosmos](#cosmos-messages)
+- [Wallets and Signers](#connecting-with-wallets-and-signing-messages)
+    - [Stargate Client](#initializing-the-stargate-client)
+    - [Creating Signers](#creating-signers)
+    - [Broadcasting Messages](#broadcasting-messages)
+- [Advanced Usage](#advanced-usage)
+- [Developing](#developing)
+- [Credits](#credits)
+
+## Usage
+### RPC Clients
+
+```js
+import { agoric } from '@agoric/cosmic-proto';
+
+const { createRPCQueryClient } = agoric.ClientFactory; 
+const client = await createRPCQueryClient({ rpcEndpoint: RPC_ENDPOINT });
+
+// now you can query the cosmos modules
+const balance = await client.cosmos.bank.v1beta1
+    .allBalances({ address: 'agoric1addresshere' });
+
+// you can also query the agoric modules
+const swingsetParams = await client.agoric.swingset.params()
+```
+
+### Composing Messages
+
+Import the `agoric` object from `@agoric/cosmic-proto`. 
+
+```js
+import { agoric } from '@agoric/cosmic-proto';
+
+const {
+    installBundle,
```

Comment:
nit: 
```suggestion
    installBundle,
    provision,
    walletAction,
    walletSpendAction,
```

There is also `deliverInbound` which could be included, though I'm not sure what it's mean to be used for.

Reply:
I see the README as illustrative. The code is always the source of truth. I think adding those will set the wrong expectation.